R/lasso-deployment.R

```diff
@@ -35,27 +33,28 @@
 #' ## 1. Loading data and packages.
 #' ptm <- proc.time()
 #' library(healthcareai)
-#'
+#' 
 #' # setwd('C:/Yourscriptlocation/Useforwardslashes') # Uncomment if using csv
 #' 
 #' # Can delete this line in your work
 #' csvfile <- system.file("extdata", 
 #'                        "HCRDiabetesClinical.csv", 
 #'                        package = "healthcareai")
-#'
+#' 
 #' # Replace csvfile with 'path/file'
 #' df <- read.csv(file = csvfile, 
 #'                header = TRUE, 
 #'                na.strings = c("NULL", "NA", ""))
-#'
-#' head(df)
-#' str(df)
+#' 
+#' df$PatientID <- NULL # remove this column
+#' 
+#' 
+#' dfDeploy <- df[951:1000,]
```

Comment:
Need to split into training and validation sets:

On line 61 we use the whole dataset to train and test a model and then later on line 78 we use a subset of that whole dataset to validate the model.  Need to make sure not to introduce future data.